README.md

```diff
@@ -118,6 +124,16 @@ version installed, you can replace `./gradlew` with `gradle` in the following st
    2d. Compile and run signal-cli:
 
         ./gradlew run --args="--help"
+3. Now you can copy the distribution that you just build, with:
+```   
+        cd build/distributions
+        # Unpack the compressed .tar file to the /opt directory.
+        sudo tar xf signal-cli-0.*.*.tar -C /opt
+        # Move the unpacked distribution into the /usr/local/bin/ directory
+        sudo ln -sf /opt/signal-cli-0.*.*/bin/signal-cli /usr/local/bin/
```

Comment:
The same instructions are already given above (https://github.com/AsamK/signal-cli#install-system-wide-on-linux), I don't think it needs to be in the README twice